server/plugin.go

```diff
@@ -4,91 +4,143 @@
 package main
 
 import (
-	"crypto/subtle"
+	"crypto/rsa"
 	"fmt"
-	"net/http"
+	"path/filepath"
 	"sync"
+	"text/template"
+
+	"github.com/pkg/errors"
 
-	"github.com/mattermost/mattermost-server/model"
 	"github.com/mattermost/mattermost-server/plugin"
 )
 
+const (
+	JIRA_USERNAME = "Jira Plugin"
+	JIRA_ICON_URL = "https://s3.amazonaws.com/mattermost-plugin-media/jira.jpg"
```

Comment:
I would like to see this being provided via the plugin. Mind creating a ticket for this? E.g. with https://github.com/mattermost/mattermost-server/pull/10476.